app/models/foreman_docker/service/containers.rb

```diff
@@ -0,0 +1,34 @@
+module ForemanDocker
+  module Service
+    class Containers
+      def self.start_container!(wizard_state)
+        container = ActiveRecord::Base.transaction do
+          container = Container.new(wizard_state.container_attributes) do |r|
+            # eagerly load environment variables
+            state = DockerContainerWizardState.includes(:environment => [:environment_variables])
+              .find(wizard_state.id)
+            state.environment_variables.each do |environment_variable|
+              r.environment_variables.build :name     => environment_variable.name,
+                                            :value    => environment_variable.value,
+                                            :priority => environment_variable.priority
+            end
+          end
+          Taxonomy.enabled_taxonomies.each do |taxonomy|
+            container.send(:"#{taxonomy}=", wizard_state.preliminary.send(:"#{taxonomy}"))
+          end
+          container.save!
+          container
+        end
+
+        # destroy_wizard_state(wizard_state)
```

Comment:
I left this commented out while I was developing this feature - should NOT be commented normally